### PR TITLE
APEXMALHAR-2526 FunctionOperator fails to resolve class from context class loader.

### PR DESCRIPTION
**Pull request by tweise**

### CODE REVIEW

**Comment by tweise** [comment]
test this please

**Comment by tweise** [comment]
test this please

**Comment by tweise** [comment]
@davidyan74 please have a look

File: library/src/main/java/org/apache/apex/malhar/lib/utils/KryoJavaSerializer.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.KryoException;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.serializers.JavaSerializer;
+import com.esotericsoftware.kryo.util.ObjectMap;
+
+/**
+ * Extension of {@JavaSerializer} for consistent class loading behavior.
+ *
+ * <p>This class can be removed after upgrade to a Kryo release > 4.0
```
Comment (davidyan74) [inline library/src/main/java/org/apache/apex/malhar/lib/utils/KryoJavaSerializer.java:35]: Should we file a JIRA so that we don't lose track of this?

**Review by davidyan74 — APPROVED** [review]

File: library/src/main/java/org/apache/apex/malhar/lib/utils/KryoJavaSerializer.java (hunk 1 above)
Comment (tweise) [reply library/src/main/java/org/apache/apex/malhar/lib/utils/KryoJavaSerializer.java:35]: Done: https://issues.apache.org/jira/browse/APEXCORE-768

**Pull request merged**